### ISSUE TITLE
Suggestion: Move MAG bolt nuke up in the rotation (version 2024-04-16)

### ISSUE THREAD

**Issue by JoeBostic** [DerpleMQ2/rgmercs#46]
The MAG class has the issue where most nuke spells can be cast over and over again since the cooldown is so fast. This means that a "rotation" is really just casting the first spell in the rotation list over and over. If the FireBolt line was moved up in the rotation, then the MAG would cast it instead of the lesser efficient nukes it has in slot #1 of the rotation. I was able to move the rotation accordingly in LUA as a test and it works great. A simple change in the "DPS" rotation list is all that is needed.

**Comment by DerpleMQ2** [comment]
Rotations don't work this way. They keep state between casts and only reset to step 1 during downtime. I am not sure you are actually seeing what you think you are - can you give some more details?

Are you casting it over and over due to going into downtime?

There should be a green button that moves down the rotation as it executes in the UI to give you a better idea of what its running.

**Comment by JoeBostic** [comment]
I've been watching my mage closely. In combat it calls "Malaise" (debuf rotation spell) and then casts "Bolt of Flame" over and over. It never casts the other nukes unless it gets to very low mana and Bolt of Flame can't be cast. FYI, this is after I moved Bolt of Flame to the top of the rotation. This is actually what I want to happen. Bolt of Flame is efficient and casting it over and over is best if mana permits.

**Comment by JoeBostic** [comment]
In a controlled experiment where I turned off Malo casting. Attacking one mob it cast the bolt spell twice, mob died. Another test where two mobs were in combat, it cast the bolt spell 4 times, both mobs dead. It is working great from what I want it to do now that I moved bolt to top of dps rotation.  Not really sure I would want it to cast each spell once in the rotation list before coming back to the bolt spell.

![image](https://github.com/DerpleMQ2/rgmercs/assets/54600963/697018bd-4a4c-4010-8804-50e0bcdc50db)
